libs/OpenShiftPipelineAPI.py

```diff
@@ -0,0 +1,146 @@
+import subprocess
+from json import JSONDecodeError
+import requests
+import os
+import json
+import time
+from robotlibcore import keyword
+
+
+class OpenShiftPipelineAPI:
+
+    def __init__(self):
+        self.project = "ods-ci-pipeline"
+        self.route = ''
+        self.sa_token = None
+
+    @keyword
+    def fetch_runs(self):
```

Comment:
I think this method should be renamed to "authenticate" or "login", as is the only one setting the credentials and is really not returning a list of runs

How are you going to test with different user credentials? (This is probably something to discuss in a meeting and not here)